client/components/environment-badge/style.scss

```diff
@@ -83,9 +83,10 @@
 	}
 }
 
-@include breakpoint( '<960px' ) {
+//TODO: DO NOT MERGE
```

Comment:
noting this `TODO` and that it still exists

Reply:
I think this might be what "Remove the CSS hiding the environment badge" in the pre-merge checklist is referring to?

Reply:
exactly the placement was just driving me nuts during development :D Don't worry, I'll remove it.